pkgs/development/python-modules/brian2/default.nix

```diff
@@ -15,22 +16,22 @@
 
 buildPythonPackage rec {
   pname = "brian2";
-  version = "2.7.0";
+  version = "2.7.1";
   pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-d9GDWp8CGIjeprWf4TtchVd36gmo36HBRkBOLaRXbpo=";
+    hash = "sha256-mp1xo6ooYm21s6FYcegQdsHmVgH81usV9IfIM0GM7lc=";
   };
 
-  build-system = [ setuptools ];
+  build-system = [ setuptools-scm ];
 
   dependencies = [
     cython
     jinja2
-    numpy
+    numpy_2
```

Comment:
Please don't propagate `numpy_2`.

> Only unversioned attributes (e.g. pydantic, but not pypdantic_1) can be included in dependencies, since due to PYTHONPATH limitations we can only ever support a single version for libraries without running into duplicate module name conflicts.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines